### ISSUE TITLE
Is the project accepting new maintainers?

### ISSUE THREAD

**Issue by jpmckinney** [scrapy/scrapyd-client#81]
The PRs are fairly simple and repetitive, and the issues are a mix of support requests (no bug/feature to add) or small changes.

We ([Open Contracting Partnership](https://www.open-contracting.org)) use scrapyd-client daily to deploy new spiders to a scrapyd instance ([our Scrapy project](https://github.com/open-contracting/kingfisher-collect)), so we're interested in this library at least being interoperable with the latest version of Scrapy (which it is not now).

I am the maintainer of many PyPi projects, notably [csvkit](https://csvkit.readthedocs.io/en/latest/), which is [many](https://pypistats.org/packages/csvkit) [times](https://pypistats.org/packages/scrapyd-client) more popular than this library, in terms of downloads, frequency of issues reported, etc.

**Comment by Gallaecio** [comment]
Thanks for stepping up! I’ve just sent you an invitation to the @scrapy/scrapyd-maintainers team.

Please, feel free to ask me or @pawelmhm any question you have. @pawelmhm is more familiar with scrapyd itself, and might be able to help out with actual code, whereas I can help with administration things (e.g. setting up a PyPI secret for a GitHub Actions workflow to publish the next release).

**Comment by jpmckinney** [comment]
@Gallaecio Great! I've made a PR to use GitHub Actions. I'll make another PR to copy-over the Travis configuration that would deploy to PyPI. Otherwise, I'm currently going through old PRs and issues that can be closed without code changes.

**Comment by pablohoffman** [comment]
Thanks for the offer @jpmckinney and welcome onboard! Feel free to tag me if you have any question, I built Scrapyd original code and may have the context for certain decision. More than happy to help!

**Comment by jpmckinney** [comment]
Thank you, all, for the warm welcome! I've done a bunch of issue triage for scrapyd-client and scrapyd, and scrapyd-client is just about ready for release.

**Comment by pablohoffman** [comment]
Awesome @jpmckinney!. Do you have all you need to make the release?

**Comment by jpmckinney** [comment]
The PR is reviewed. I’m just waiting on a TEST_PYPI_TOKEN secret, to test that each PR can build against https://test.pypi.org

I could move this to a separate PR, or create my own account on Test PyPI, but I figured I’d wait for an account that Scrapy developers can share.
